files/image_config/interfaces/interfaces.j2

```diff
@@ -9,13 +9,6 @@
 # The loopback network interface
 auto lo
 iface lo inet loopback
-# Use command 'ip addr list dev lo' to check all addresses
```

Comment:
@tylerlinp @prsunny Any reason why the approach used for mgmt i/f being in the vrf can't be followed for lo? Not even sure that lo should be allowed to be in a vrf. How will redis work if lo is in a vrf?

Reply:
> @tylerlinp @prsunny Any reason why the approach used for mgmt i/f being in the vrf can't be followed for lo? Not even sure that lo should be allowed to be in a vrf. How will redis work if lo is in a vrf?

All loopback interfaces map to device lo, that cause conflict when loopbacks belong to differents vrfs. I think lo shouldn't be add to a vrf, even though kernel don't restrict doing so. Some process which communicates locally may be affected. It is good to add a device for every loopback interface(LoopbackXXX), then lo keep original function. Then LoopbackXXX is add vrf/ip just like other l3 interface.

Reply:
Yes I understand the issue. However, you are taking away the ability to further statically configure lo with this change.

Reply:
> Yes I understand the issue. However, you are taking away the ability to further statically configure lo with this change. The right approach to follow would be to leave lo configuration where it is and make sure other loopbacks don't map to the same loopback device.

What configuration do you mean? The ip addresses only config to kernel? If not add to ASIC, I think the addresses except 127.x.x.x & ::1 would be useless. If truely needed, maybe should filter configuration like LOOPBACK_INTERFACE|lo|1.1.1.1/32 and then add to interfaces.